tasks/addon/root/routes.js.underscore

```diff
@@ -11,30 +11,35 @@
 // the License.
 
 define([
-  "app",
-  "api",
-  "addons/{%= name.toLowerCase() %}/resources"
+  'app',
+  'api',
+  'addons/{%= name.toLowerCase() %}/resources',
+  'addons/{%= name.toLowerCase() %}/components.react'
 ],
-function (app, FauxtonAPI, {%= name %}) {
-  var {%= name.substr(0, 1).toUpperCase() + name.substr(1)%}RouteObject = FauxtonAPI.RouteObject.extend({
-    layout: "with_sidebar",
 
-    crumbs: [
-    ],
+function (app, FauxtonAPI, {%= name %}, Components) {
 
+  var {%= name.substr(0, 1).toUpperCase() + name.substr(1)%}RouteObject = FauxtonAPI.RouteObject.extend({
+    layout: 'one_pane',
+    roles: [],
     routes: {
+      'example/addon': 'helloWorld'
```

Comment:
i was suprised by the route when testing it...

how about naming a route like: 

`foo: 'helloWorld'` for a module that is named `foo`

```js
routes: {
  '{%= name %}': 'helloWorld'
```

i think?